CAM2CameraDatabaseAPIClient/client.py

```diff
@@ -619,6 +619,67 @@ def camera_by_id(self, cameraID):
                 raise InternalError()
         return Camera.process_json(**response.json())
 
+    def camera_by_legacy_id(self, legacy_cameraID):
+        """
+        A method to get a camera object by using camera's legacy ID
+
+        Parameters
+        ----------
+        legacy_cameraID : str
+            legacy_cameraID of the camera in the database.
+
+        Returns
+        -------
+        :obj:`Camera`
+            A camera object.
+
+        """
+        if self.token is None:
+            self._request_token()
+        url = Client.base_URL + "cameras/legacy/" + legacy_cameraID
+        header = self.header_builder()
+        response = self._check_token(response=requests.get(url, headers=header),
+                                     flag='GET', url=url)
+
+        if response.status_code != 200:
+            if response.status_code == 401:
+                raise AuthenticationError(response.json()['message'])
+            elif response.status_code == 404:
+                raise ResourceNotFoundError(response.json()['message'])
+            elif response.status_code == 403:
+                raise AuthorizationError(response.json()['message'])
+            elif response.status_code == 422:
+                raise FormatError(response.json()['message'])
+            else:
+                raise InternalError()
+        return Camera.process_json(**response.json())
+
+    def camera_by_list_id(self, cameraIDList, legacy_cameraIDList):
```

Comment:
I feel it might be more intuitive to have method declaration as `camera_by_list_id(self, cameraID_list=None, legacy_cameraID_list=None)`. First, we might want not to use camel toe for variable names for consistency. Secondly, it will be more frequent when a user only has a list of one type of cameraID, whether cameraID or legacy_cameraID. user can call `client.camera_by_list_id( cameraID_list = ['a', 'b', 'c'])` instead of having to specify that `client.camera_by_list_id( ['a', 'b', 'c'], None)` where human errors in the order of lists might happen.

Reply:
done